packages/frontend-2/components/settings/workspaces/automation/Functions/Functions.vue

```diff
@@ -0,0 +1,117 @@
+<template>
+  <div class="flex flex-col gap-4">
+    <SettingsSectionHeader
+      subheading
+      title="Functions"
+      :buttons="[
+        {
+          props: {
```

Comment:
too much code for the template IMO, i would move this to a computed (w/ a LayoutDialogButton[] type expectation, for easier typing/autocomplete)

Reply:
Ended up being a slightly different type, but did the same idea